src/Public/Update-Package.ps1

```diff
@@ -34,7 +34,7 @@
     # The following script is used to update the package from the github releases page.
     # After it defines the 2 functions, it calls the Update-Package.
     # Checksums are automatically calculated for 32 bit version (the only one in this case)
-    import-module au
+    import-module Chocolatey-AU
```

Comment:
Is it possible for backwards compatibility reasons to keep an import alias for `AU` as well? Or do we only have the option to use `Chocolatey-AU`?

Reply:
Unfortunately I do not believe you can create aliases for modules.

Reply:
I think we should have a clear separation - if you install `chocolatey-au` you need to be intentional about that. We will be doing things differently. It has to be intentional.